common/modules/gcp-stackdriver-monitoring/uptime_checks.tf

```diff
@@ -0,0 +1,26 @@
+locals {
```

Comment:
Why is the `uptime_checks.tf ` being created here at all? I don't think there's anything to monitor in common, or what am i misisng?